AnkiDroid/src/test/java/com/ichi2/libanki/CardTest.kt

```diff
@@ -337,4 +337,40 @@ class CardTest : RobolectricTest() {
         assertEquals("(filtered)", c.nextDue())
         assertEquals("((filtered))", c.dueString)
     }
+
+    @Test
+    fun testAvgEaseOfNote() {
+        // basic case: no cards are new
+        val note = addNoteUsingModelName("Cloze", "{{c1::Hello}}{{c2::World}}{{c3::foo}}{{c4::bar}}", "extra")
+        // factor for cards: 3000, 1500, 1000, 750
+        var i = 1
+        for (card in note.cards()) {
+            card.type = Consts.CARD_TYPE_REV
+            card.factor = 3000 / i++
+            card.flush()
+        }
+        // avg ease = (3000/10 + 1500/10 + 100/10 + 750/10) / 4 = [156.25] = 156
+        var avgEase = 156
+        assertEquals(avgEase, note.firstCard().avgEaseOfNote())
```

Comment:
Remove the variable and supply it directly in the assertion
```suggestion
        assertEquals(156, note.firstCard().avgEaseOfNote())
```